src/charm.py

```diff
@@ -135,6 +135,8 @@ def _on_upgrade_charm(self, _):
         self._configure()
 
     def _on_loki_pebble_ready(self, _):
+        self._ensure_alert_rules_path()
+        self._regenerate_alert_rules()
```

Comment:
`_ensure_alert_rules_path` return a boolean.

Why do not we do the same we are doing in `_loki_push_api_alert_rules_changed` method?

```suggestion
        if self._ensure_alert_rules_path():
            self._regenerate_alert_rules()
```